analyzers/src/SonarAnalyzer.CSharp/Rules/TestMethodShouldContainAssertion.cs

```diff
@@ -144,7 +148,7 @@ private static bool IsTestIgnored(IMethodSymbol method)
                 ?.ApplicationSyntaxReference.GetSyntax() as AttributeSyntax;
 
             return factAttributeSyntax?.ArgumentList != null
-                && factAttributeSyntax.ArgumentList.Arguments.Any(x => x.NameEquals.Name.Identifier.ValueText == "Skip");
+                   && factAttributeSyntax.ArgumentList.Arguments.Any(x => x.NameEquals.Name.Identifier.ValueText == "Skip");
```

Comment:
This wasn't needed, we follow 4 tab indent
```suggestion
                 && factAttributeSyntax.ArgumentList.Arguments.Any(x => x.NameEquals.Name.Identifier.ValueText == "Skip");
```